statsmodels/iolib/foreign.py

```diff
@@ -1160,7 +1160,7 @@ def savetxt(fname, X, names=None, fmt='%.18e', delimiter=' '):
             import gzip
             fh = gzip.open(fname, 'wb')
         else:
-            fh = file(fname, 'w')
+            fh = open(fname, 'w')
```

Comment:
Good point, I corrected this. There is now a slightly more general 'fileopen' module to create the right objects.